### PR TITLE
Guarin lig 3094 update advanced concepts in ssl docs

### PR DESCRIPTION
**Pull request by guarin**
## Changes

* Update advanced concepts docs
* Replace references to collate functions with transforms
* General cleanup

### CODE REVIEW

File: docs/source/getting_started/advanced.rst [hunk 1]
```diff
@@ -380,11 +395,11 @@ not have noticed the representation collapse in the first run and continued
...
 
 
-Extracting specific Video Frames
+Extracting Specific Video Frames
 --------------------------------
 
 When working with videos, it is preferred not to have to extract all 
-the frames beforehand. With lightly we can not only subsample the video 
+the frames beforehand. With Lightly we can not only subsample the video 
 to find interesting frames for annotation but also extract only these frames.
```
Comment (MalteEbner) [inline docs/source/getting_started/advanced.rst:404]: Can we also remove this section completely?
It does not make that much sense to me:
- Lightly is not designed to be a  video frame loader
- The code looks like is if from an outdated workflow: `read the list of filenames (e.g. from the Lightly Docker output)`

**Review by MalteEbner — COMMENTED** [review]
This is fine, only a few points.

File: docs/source/getting_started/advanced.rst [hunk 2]
```diff
@@ -64,62 +63,116 @@ Some interesting papers regarding invariances in self-supervised learning:
...
+
+    from lightly.transforms import SimCLRTransform
+
+    transform = SimCLRTransform(
+        input_size=128,   # resize input images to 128x128 pixels
+        cj_prob=0.0,      # disable color jittering
+        rr_prob=0.5,      # apply random rotation by 90 degrees with 50% probability
+    )
+
+  .. note:: You can disable the augmentations by either setting the probability to `0.0`
```
[the remark this answers is not in the document]
Comment (guarin) [reply docs/source/getting_started/advanced.rst:112]: Not all augmentations have a probability. For example random cropping is always applied. The only way to disable it is to set `min_scale=1.0`.

File: docs/source/getting_started/advanced.rst (hunk 1 above)
Comment (guarin) [reply docs/source/getting_started/advanced.rst:404]: I agree, let's remove it.

File: docs/source/getting_started/advanced.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (MalteEbner) [reply docs/source/getting_started/advanced.rst:112]: Ok, let's keep it.

**Review by MalteEbner — APPROVED** [review]
LGTM

**Pull request merged**